data/config.dist.json

```diff
@@ -5,6 +5,6 @@
     "array", "string", "int", "float", "bool", "iterable", "callable", "void"
   ],
   "php-core-extensions" : [
-    "Core", "date", "pcre", "Reflection", "SPL", "standard"
+    "Core", "date", "pcre", "Phar", "Reflection", "SPL", "standard"
```

Comment:
sure, I put it as RFC actually, modifying it from web ui actually :D

question - actually I would like to have `HHVM_VERSION` in my code. I check it conditionally (we already have ticket raised by @Ocramius to have weak checker, so let us not focus on it in this PR), so I want to add `HHVM_VERSION` to exception list.
Currently, I need to create config file which will have all default values copy-pasted from this file plus values that I will provide. So whenever sth was modified in main repo, I don't take advantages from it. Maybe it would be better to have some kind of "override" vs "add" whitelist items ? WDYF?